### PR TITLE
added function button

### PR DESCRIPTION
**Pull request by tanjahennis**
I've added the LEARN MORE button to link to the UWV scholingsvoucher site. 
screen shot 2016-10-27 at 12 28 05
 

On app/views/static_pages/home.html.slim line 44-71 deletions are changes that are no part of this commit, I'm reinserting them for the next bit where I will be working on the 'Visit us' part of the homepage - to be checked for that commit. 
Same goes for the changes on app/assets/stylesheets/materialized-components/_redesign_visit.sass.

So basically only the green line 44 on app/views/static_pages/home.html.slim is relevant. Hope this is sort of clear 🙈

### CODE REVIEW

**Comment by miriamtocino** [comment]
It was not really necessary to do this one, but I now you want to keep your pull requests clean :smile:

Go for it!
🚢 🇮🇹
